packages/gatsby-telemetry/src/send.js

```diff
@@ -0,0 +1,15 @@
+const Telemetry = require(`./telemetry`)
+const instance = new Telemetry()
+
+const flush = _ => {
+  instance
+    .sendEvents()
+    .then(e => {
```

Comment:
Do you need this `.then` with a `.catch` below it?

Reply:
Indeed, `.catch` should suffice.